go.mod

```diff
@@ -3,4 +3,5 @@ module github.com/atlassian/terraform-provider-artifactory
 require (
 	github.com/atlassian/go-artifactory/v2 v2.1.1
 	github.com/hashicorp/terraform v0.11.13
+	golang.org/x/net v0.0.0-20190324223953-e3b2ff56ed87 // indirect
```

Comment:
We can bump go-artifactory to v2.2.0 but we should not be adding indirect dependencies

Reply:
Yeah I wasn't sure about that indirect dependency, it popped up automatically. If it's not required I'll delete it